packages/v4/src/content/contribute/design-guidelines/design-guidelines.md

```diff
@@ -0,0 +1,204 @@
+
+<img src="./image-4-organization.png"  alt="Sketch artboards with corresponding images"  width="1600"/> 
+
+## Step 8: Export and add image assets 
+
+To ensure high-quality images in the final documentation, export sketch images as PNG x2.  
+
+<img src="./image-5-export.png"  alt="Sketch export screen"  width="474"/> 
+
+Exporting as PNG x2 ensures better image quality, but it also doubles physical size (WxL). To maintain good image quality while restricting image width to the original artboard width, add images to Markdown using the following HTML tag: 
+
+`<img src="./filename.png"  alt="accessibility text describing the image"  width="width of the sketch artboard"/>`  
+
+The image should display in the visual designer of the Markdown tool you’re using. If the example does not display, there might be a problem with your code. Check that the element is correct and did not turn into normal quotations while copy/pasting. 
+
+## Step 9: Stage and commit changes 
+
+Once you’re finished making changes, stage them in Visual Studio Code:  
+
+1. When you've completed all changes, click on the numbered bubble in the toolbar’s code icon.  
+
+<img src="./image-6-bubble.png"  alt="changes bubble in Microsoft Visual Studio Code"  width="96"/>  
+
+2. In the **Changes** tab, right-click and select **Stage all changes**. 
+
+<img src="./image-7-stage.png"  alt="stage changes option in Microsoft Visual Studio Code"  width="786"/> 
+
+3. Return to your Terminal application to commit the changes: 
+
+    a. Type `git status` and press **Enter**. This should list the same files that were listed in your “changes” tab in VS. If there aren’t any errors, continue to the next step. 
+    
+    b. Type  `git commit -m ‘title of your PR’` and press **Enter**. You can edit this title later in GitHub if necessary, but it must be placed between single quote marks ( ‘ ) for now. Be sure to type these single quote marks—copy/pasting them from this page won’t register correctly. 
+    
+    c. Type `git push origin --set-upstream [name of your PR]` and press **Enter**. 
+
+## Step 10: Create a pull request 
+
+On GitHub, create a pull request to submit your changes: 
+
+1. Go to the patternfly-org repo pulls and click **Compare & pull request**.  
+
+<img src="./image-8-yellow-bar.png"  alt="Compare & pull request option in GitHub"  width="512"/> 
+
+2. On the resulting page, make sure that the arrow points towards the base repository and away from the head repository. 
+
+<img src="./image-9-pr.png"  alt="creating a pull request screen in GitHub"  width="1600"/> 
+
+3. Change your PR name and/or add a comment to your PR if you need to.
+4. Attach all of your image files to this PR so that they can be updated in the original document, merged back into the main branch, and uploaded back to Sketch Cloud for future access.
+5. Tag [mmenestr](https://github.com/mmenestret/mmenestret.github.io) to initiate a final review. 
```

Comment:
Link to my GitHub is wrong -- it says "mmenestret" but should just be "/mmenestr" just like the tagged text

Reply:
Fixed!